chain/index/api.go

```diff
@@ -285,33 +286,22 @@ func (si *SqliteIndexer) verifyIndexedData(ctx context.Context, ts *types.TipSet
 }
 
 func (si *SqliteIndexer) backfillMissingTipset(ctx context.Context, ts *types.TipSet) (*types.IndexValidation, error) {
-	// backfill the tipset in the Index
-	parentTs, err := si.cs.GetTipSetFromKey(ctx, ts.Parents())
-	if err != nil {
-		return nil, xerrors.Errorf("failed to get parent tipset at height %d: %w", ts.Height(), err)
-	}
-
 	executionTs, err := si.getNextTipset(ctx, ts)
 	if err != nil {
 		return nil, xerrors.Errorf("failed to get next tipset at height %d: %w", ts.Height(), err)
 	}
 
 	backfillFunc := func() error {
 		return withTx(ctx, si.db, func(tx *sql.Tx) error {
-			if err := si.indexTipsetWithParentEvents(ctx, tx, ts, executionTs); err != nil {
-				return xerrors.Errorf("error indexing (ts, executionTs): %w", err)
-			}
-
-			if err := si.indexTipsetWithParentEvents(ctx, tx, parentTs, ts); err != nil {
-				return xerrors.Errorf("error indexing (parentTs, ts): %w", err)
-			}
-
-			return nil
+			return si.indexTipsetWithParentEvents(ctx, tx, ts, executionTs)
 		})
 	}
 
 	if err := backfillFunc(); err != nil {
-		return nil, xerrors.Errorf("failed to backfill tipset: %w", err)
+		if ipld.IsNotFound(err) {
+			return nil, xerrors.Errorf("failed to backfill tipset at epoch %d: chain store does not contain data", ts.Height())
```

Comment:
```suggestion
			return nil, xerrors.Errorf("failed to backfill tipset at epoch %d: chain store does not contain data: %w", ts.Height(), err)
```

It might be wise to still return the original error, just in case there's something unexpected going on and we need to debug?